supervisor/misc/filter.py

```diff
@@ -38,25 +38,39 @@ def filter_data(coresys: CoreSys, event: dict, hint: dict) -> dict:
     if coresys.core.state in (CoreStates.INITIALIZE, CoreStates.SETUP):
         return event
 
+    # List installed addons
+    installed_addons = [
+        {"slug": addon.slug, "repository": addon.repository, "name": addon.name}
+        for addon in coresys.addons.installed
+    ]
+
     # Update information
     event.setdefault("extra", {}).update(
```

Comment:
We should start using `contexts` instead.

It will render its own boxes (according to the founder of sentry, that will be made searchable in the future as well).

I've just applied it locally on the Core for testing:

![image](https://user-images.githubusercontent.com/195327/90311514-da5e6b80-defb-11ea-9e5e-6feb9b676599.png)


```py
        event.setdefault("contexts", {}).update(
            {
                "system": system_info,
                "integrations": {
                    "active": "\n".join(sorted(integrations)),
                    "custom_components": "\n".join(sorted(custom_components)),
                },
            }
        )
```

Reply:
I like that!